scripts/plot_dataframe.py

```diff
@@ -11,24 +11,221 @@
 --timerange: specify what timerange of data to use.
 -l / --live: Live, to download the latest ticker for the pair
 """
+import datetime
 import logging
 import sys
 from argparse import Namespace
-
 from typing import List
 
+import plotly.graph_objs as go
```

Comment:
should this be just `import plotly.graph_objs as graph_objs`to make it more readable?